conversion_helpers.go

```diff
@@ -41,7 +80,7 @@ type QueueStateSnapshot struct {
 	Timestamp time.Time `json:"timestamp"`
 }
 
-func toQueueStateSnapshot(s *asynq.QueueInfo) *QueueStateSnapshot {
+func (t *transformer) toQueueStateSnapshot(s *asynq.QueueInfo) *QueueStateSnapshot {
```

Comment:
question: What's the reason for changing these helper functions to methods?

Reply:
https://github.com/hibiken/asynqmon/pull/157#discussion_r719647373